R/teardown.R

```diff
@@ -16,7 +16,7 @@ teardown_env$queue <- list()
 #' \dontrun{
 #'
 #' tmp <- tempfile()
-#' setup(writeLines(tmp, "some test data"))
+#' setup(writeLines(con = tmp, "some test data"))
```

Comment:
I think it would be better to just flip the argument order here. I always forget which comes first 😞

Reply:
I follow your comment.